AmplifyPlugins/Auth/Tests/AWSCognitoAuthPluginUnitTests/TaskTests/ClientBehaviorTests/AuthenticationProviderDeleteUserTests.swift

```diff
@@ -110,16 +112,17 @@ class AuthenticationProviderDeleteUserTests: BasePluginTest {
     ///    - I should get a .service error with .network as underlying error for the first call
     ///    - I should get a valid response for the second call
     ///
-    func testOfflineDeleteUserAndRetry() async {
+    func testOfflineDeleteUserAndRetry() async throws {
+        // TODO: How are client side retry errors now modeled?
```

Comment:
same comment here, does TODO need to be resolve?

Reply:
addressed in [a76284a](https://github.com/aws-amplify/amplify-swift/pull/3248/commits/a76284a55880354a9c5a1b1f289fecf62155cb62)